app/dashboard/templates/profiles/profile.html

```diff
@@ -234,15 +234,66 @@
                       </div>
                     </div>
                   </div>
-                </div>
-                {% endif %}
+                </div> {% endif %}
               </div>
             {% endfor %}
-          </div>
-        </div>
-      {% endif %}
-    {% endif %}
-    </div>
+		{% else %}
+			<div class="row justify-content-center mt-2">
+				<img width="240px" height="225px" src="{% static "v2/images/tree.png" %}"/>
+			</div>
+			<div class="mt-2 text-center" style="color: #666666; font-size: 16px">
```

Comment:
could you use classes `font-subheader` / `font-body` / `font-caption` as opposed to hardcoding the font-size